tests/scripts/run_external_tests.sh

```diff
@@ -27,6 +27,7 @@ cd "${scripts_dir}"
 
 test_args=(
   "${common_test_args[@]}"
+  "--experimental_enable_bzlmod=${USE_BZLMOD:-false}"
```

Comment:
```suggestion
  "--enable_bzlmod=${USE_BZLMOD:-false}"
```
with Bazel 6 and higher.

Reply:
5.1.0 is still being tested in CI (but only with bzlmod disabled) so this is why this is still using the --experimental version of this flag.